docs/user_guide/metrics.md

```diff
@@ -100,6 +100,19 @@ Count*. The count metrics are illustrated by the following examples:
 |              |Execution Count |`nv_inference_exec_count` |Number of inference batch executions (see [Inference Request Metrics](#inference-request-metrics), does not include cached requests)|Per model|Per request|
 |              |Pending Request Count |`nv_inference_pending_request_count` |Number of inference requests awaiting execution by a backend. This number is incremented when a request is enqueued to the server (`TRITONSERVER_ServerInferAsync`) and is decremented when a backend is about to start executing the request. More details can be found below. |Per model|Per request|
 
+#### Failure Count is further classified into four distinct types:
+
+|Sub-Category      |Sub-Metric | Sub-Metric Name |Description |
```

Comment:
I think this can be condensed to 2 columns for now, something like:
---

|Failed Request Reason | Description |
|---|---|
| REJECTED | Number of ... |
| CANCELLED | Number of ... |
| BACKEND | Number of ... |
| OTHER | Number of ... |